spring-cloud-function-samples/function-sample/build.gradle

```diff
@@ -45,6 +45,5 @@ dependencyManagement {
 
 dependencies {
 	compile('org.springframework.cloud:spring-cloud-starter-function-web')
-	compile('org.springframework.cloud:spring-cloud-function-compiler')
```

Comment:
This is not used in the sample. However, somewhat unrelated point...

This sample module is not specified in the base pom and therefore not included in the build commands (effectively disabled/inactive/etc..).
1. Is this intentional?
2. the docs "getting started" point to this sample (but it does not build by default)
3. should we delete this sample altogether and instead point to the`function-sample-pojo` in the "getting started" docs?

Reply:
i think we can rework it, clean it etc since docs point to this. Feel free to create a separate issue for that

Reply:
Will do. I will create an issue to do this + the removal of the task sample.

Reply:
https://github.com/spring-cloud/spring-cloud-function/issues/812